fluidefi/fluidefi-caspernet-aggregator-tools
# Block hour checker

**Summary of change:**
- Added a new micro-service, block hours checker. 

**List of tasks:** 

- [x] create fixtures.

**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/block-hours-checker`
```bash
git checkout feature/block-hours-checker
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm block-hour-checker test has passed.
- Run example.
